aiida/cmdline/commands/cmd_quicksetup.py

```diff
@@ -68,11 +69,12 @@ def quicksetup(profile_name, only_config, set_default, non_interactive, backend,
     # be named test_...
     import getpass
     osuser = getpass.getuser()
-    dbname = db_name or profile_name + '_' + osuser
+    aiida_base_dir_hash = hashlib.md5(AIIDA_CONFIG_FOLDER).hexdigest()
+    dbname = db_name or profile_name + '_' + osuser + '_' + aiida_base_dir_hash
 
     # default database user name is aiida_qs_<login-name>
     # default password is random
-    dbuser = db_username or 'aiida_qs_' + osuser
+    dbuser = db_username or 'aiida_qs_' + osuser + '_' + aiida_base_dir_hash
```

Comment:
I get that it makes sense to use a different DB for different config folders.
If the DB is already different, do we still need to append the hash to the username?

Reply:
In fact the username is just as important: if it already exists (from another config.json), there is no way to know the password stored in the other config.json.